README.md

```diff
@@ -373,12 +373,12 @@ A `mesh.sh` bash script exists for testing or debugging MESH. For more informati
 
 #### Fake MESH
 
-A mock implementation of the MESH API is available for local development. The latest version is in Github at
-[mattd-kainos/fake-mesh](https://github.com/jamespic/fake-mesh). _It is a fork of [jamespic/fake-mesh](https://github.com/jamespic/fake-mesh)._
+A mock implementation of the MESH API is available for local development. The latest version is in GitHub at
+[mattd-kainos/fake-mesh](https://github.com/jamespic/fake-mesh). _It is a fork of [jamespic/fake-mesh](https://github.com/jamespic/fake-mesh).
```

Comment:
Minor: This `mattd-kainos` link looks wrong.

Reply:
have found the correct link and updated